NEWS.md

```diff
@@ -1,7 +1,13 @@
 # IsoMemo 1.0.5001
 
 # corrections with name changes and corrections:
```

Comment:
Can yoy make this title smaller? Looks very important, but it isn't:
![image](https://user-images.githubusercontent.com/16759098/173818993-d4d79a1d-9b61-4e85-8f09-d9553f17588b.png)